benchopt/plotting/generate_html.py

```diff
@@ -125,18 +132,21 @@ def shape_objectives_columns_for_html(df, dataset, objective):
         if c.startswith('objective_') and c != 'objective_name'
     ]
     for column in columns:
-        df_filtered = df.query("data_name == @dataset & objective_name == @objective")
+        df_filtered = df.query(
+            "data_name == @dataset & objective_name == @objective")
```

Comment:
Nitpick on the style
```suggestion
            "data_name == @dataset & objective_name == @objective"
        )
```